src/components/richText/DefaultStyling.ts

```diff
@@ -23,6 +37,38 @@ const DefaultNodeStyling: EditorThemeClasses = {
     ol: "p-0 m-0 ml-4, list-decimal",
     listitem: "mx-8 my-2",
   },
+  code: "bg-gray-100 block text-xs m-0 mt-2 mb-2 overflow-x-auto relative p-2",
+  codeHighlight: {
```

Comment:
What do you think about organizing the styles like this:
```ts
const createStyleObject = (cases: keyof EditorThemeClasses['codeHighlight'], color) => Object.fromEntries(cases.map(rule => [rule, color]))
const attributeStyles = createStyleObject(['atrule', 'attr'], 'text-sky-700')
{ codeHighlight: {
  ...attributeStyles,  // etc
}}
```
I was thinking it may be overkill but after looking at it more I think it could help make the list of styles less overwhelming.

Reply:
Actually probably the simplest way to store things would be an object of color to different cases like
```ts
{
  'test-sky-700': ['atrule', 'attr']
}
```